subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/GradleImplDepsCompatibilityIntegrationTest.groovy

```diff
@@ -104,11 +104,17 @@ class GradleImplDepsCompatibilityIntegrationTest extends BaseGradleImplDepsInteg
 
     def "Gradle API and TestKit are compatible regardless of order #dependencyPermutations"() {
         when:
-        buildFile << applyGroovyPlugin()
+        buildFile << applyPlugins(['groovy', 'jvm-test-suite'])
```

Comment:
We don't need to explicitly apply this plugin everywhere.  It's a part of the Groovy/Scala/Java plugins